cypress/e2e/api/engage-sphere-customers.spec.cy.js

```diff
@@ -0,0 +1,256 @@
+           cy.request({
+                method: 'GET',
+                url: `${API_URL}/customers`,
+                headers: { accept: 'application/json' },
+                failOnStatusCode: false,
+            }).then(({ status, body }) => {
+                expect(status).to.eq(200);
+                const customers = body.customers;
+                expect(customers).to.be.an('array').and.not.be.empty;
+                customers.forEach((customer) => {
+                    expect(customer).to.have.all.keys(
+                        'id',
+                        'name',
+                        'employees',
+                        'industry',
+                        'contactInfo',
+                        'address',
+                        'size'
+                    );
+                });
+            });
+        });
+
+        it('GET /customers - correctly handles pagination across multiple pages', () => {
+            cy.request({
+                method: 'GET',
+                url: `${API_URL}/customers`,
+                headers: { accept: 'application/json' },
+                failOnStatusCode: false,
+            }).then(({ status, body }) => {
+                expect(status).to.equal(200);
+                expect(body.customers).to.be.an('array').with.lengthOf(10);
+                expect(body.pageInfo).to.have.all.keys('currentPage', 'totalPages', 'totalCustomers');
+                expect(body.pageInfo.currentPage).to.equal(1);
+            });
+            cy.request({
+                method: 'GET',
+                url: `${API_URL}/customers`,
+                qs: { page: 2, limit: 10 },
+                headers: { accept: 'application/json' },
+                failOnStatusCode: false,
+            }).then(({ status, body }) => {
+                expect(status).to.equal(200);
+                expect(body.customers).to.be.an('array').with.lengthOf.at.most(10);
+                expect(body.pageInfo.currentPage).to.equal(2);
+
+            });
+        });
+
+        it('GET /customers - correctly filters by customer limit', () => {
+            const limit = 15; //Maximum number of customers to be returned
+
+            cy.request({
+                method: 'GET',
+                url: `${API_URL}/customers`,
+                qs: { limit },
+                headers: { accept: 'application/json' },
+                failOnStatusCode: false,
+            }).then(({ status, body }) => {
+                expect(status).to.equal(200);
+                expect(body.customers).to.be.an('array');
+                expect(body.customers.length).to.be.at.most(limit);
+                expect(body.pageInfo).to.have.all.keys('currentPage', 'totalPages', 'totalCustomers');
+            });
+        });
+
+        it('GET /customers - filter by company size', () => {
+            const size = 'Very Large Enterprise'; // Small, Medium, Enterprise, Large Enterprise, Very Large Enterprise
```

Comment:
Que tal um comentário mais informativo, como o seguinte?

```suggestion
            const size = 'Very Large Enterprise'; // Possible sizes: Small, Medium, Enterprise, Large Enterprise, Very Large Enterprise
```

Reply:
Algum motivo para não ter aceito essa sugestão?